_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,230 @@
+* Credential passthrough, which allows Presto to authenticate with the underlying data source with 
+  credentials provided by the user running a query. We found this especially useful when dealing with
+  Google Storage in GCP or SQL databases that manage user authentication and authorization on 
+  their own.
+* Impersonation for [Hive metastore](/docs/current/connector/hive.html#hive-thrift-metastore-configuration-properties).
+* Support for reading and writing encrypted files in HDFS using Hadoop KMS.
+
+## Geospatial
+
+* New geospatial functions: 
+  [`ST_Points`](/docs/current/functions/geospatial.html#ST_Points), 
+  [`ST_Length`](/docs/current/functions/geospatial.html#ST_Length), 
+  [`ST_Area`](/docs/current/functions/geospatial.html#ST_Area), 
+  [`line_interpolate_point`](/docs/current/functions/geospatial.html#line_interpolate_point) and 
+  [`line_interpolate_points`](/docs/current/functions/geospatial.html#line_interpolate_points).
+* `SphericalGeography` type and [related functions](/docs/current/functions/geospatial.html#to_spherical_geography) 
+  to support spatial features in geographic coordinates (latitude / longitude) on spherical model of the earth.
+* Support for Google Maps Polyline format via [`to_encoded_polyline`](/docs/current/functions/geospatial.html#to_encoded_polyline)
+  and [`from_encoded_polyline`](/docs/current/functions/geospatial.html#from_encoded_polyline) functions.
+* [`geometry_from_hadoop_shape`](/docs/current/functions/geospatial.html#geometry_from_hadoop_shape) to decode geometry objects in 
+  Spatial Framework for Hadoop representation
+
+## Cloud Integration
+
+* Support for Azure Data Lake Blob and ADLS Gen2 storage.
+* Support for [Google Cloud Storage](/docs/current/connector/hive-gcs-tutorial.html).
+* Several [performance improvements](https://github.com/prestosql/presto/pull/142) for AWS S3.
+
+## CLI and JDBC Driver
+
+* Syntax highlighting and multi-line editing.
+ 
+  ![](/assets/blog/2019-review/presto-cli.jpeg)
+
+* JSON output format and improvements to CSV output format.
+* Support & stability improvements for running the CLI and JDBC driver with Java 11.
+* Improve compatibility of JDBC driver with third-party tools.
+
+## New Connectors
+
+* [Elasticsearch](/docs/current/connector/elasticsearch.html)
+* [Google Sheets](/docs/current/connector/googlesheets.html)
+* [Amazon Kinesis](/docs/current/connector/kinesis.html)
+* [Apache Phoenix]({% post_url 2019-06-04-phoenix-connector %})
+* [MemSQL](/docs/current/connector/memsql.html)
+* Apache Iceberg
+
+## Other Improvements
+
+* [Presto Docker image](https://hub.docker.com/r/prestosql/presto) that provides an out-of-the-box single node 
+  cluster with the JMX, memory, TPC-DS, and TPC-H catalogs. It can be deployed as a full cluster by 
+  mounting in configuration and can be used for Kubernetes deployments.
+* LZ4/ZSTD support for Parquet and ORC. These are the recommended compression algorithms.
```

Comment:
```suggestion
* Support LZ4 and ZSTD compression codecs in Parquet and ORC. These are the recommended compression algorithms (LZ4 for fast lightweight compression and ZSTD otherwise).
```